packages/core/src/app/checkout/mapToCheckoutProps.ts

```diff
@@ -31,6 +37,8 @@ export default function mapToCheckoutProps({
         },
     );
 
+    const walletButtonsOnTopFlag = !!features['CHECKOUT-7222.checkout_settings_styling_section'] && walletButtonsOnTop;
```

Comment:
Shall we change it to `Boolean(features['CHECKOUT-7222.checkout_settings_styling_section'])`